renovate-config.json

```
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "timezone": "Europe/Warsaw",
+  "extends": ["config:base", ":disableDependencyDashboard"],
+  "schedule": [
+    "before 5am on Monday"
+  ],
```

I suggest adding a default name for branches made by `renovate-bot` to group them easily.

```suggestion
  ],
  "branchPrefix": "deps/",
```

---

The default one it `renovate/` if we do not set anything, but deps sounds better, thanks!